tools/misbehaving-jmx-server/src/main/java/org/datadog/supervisor/App.java

```diff
@@ -0,0 +1,159 @@
+package org.datadog.supervisor;
```

Comment:
Consider if it would make more sense to just use s6 or supervisord instead of this supervisor process.
Its a little bit over-engineered at the moment and we don't actually use the "start/stop jmxserver" functionality yet. I'm leaning towards simplifying this